yesod-auth-oauth2.cabal

```diff
@@ -4,7 +4,7 @@ cabal-version: 1.12
 --
 -- see: https://github.com/sol/hpack
 --
--- hash: d3ba666a53fb63a2f059501ac04eca709a905d0dd1f806ea5a7cd64e06ef890d
+-- hash: 233909874fdbdbd71fa70c49f5a4223b4150b85d9415dbbed7fde2fff9e5ebcf
```

Comment:
I feel like this is probably a bad change?

Reply:
No it's good. It just happens any time this file's contents change, and you added a module.